src/compute/src/render/continual_task.rs

```diff
@@ -394,11 +395,10 @@ fn continual_task_sink<G: Scope<Timestamp = Timestamp>>(
     append_times: Collection<G, (), Diff>,
     as_of: Antichain<Timestamp>,
     write_handle: impl Future<Output = WriteHandle<SourceData, (), Timestamp, Diff>> + Send + 'static,
-    start_signal: StartSignal,
```

Comment:
Nit: There is a outdated comment a couple lines below that talks about "The start_signal below...", which is gone now.